Scripts/Core/EditorVR.Rays.cs

```diff
@@ -214,9 +214,17 @@ void OnProxyActiveChanged(IProxy proxy)
 				}
 			}
 
+			internal void UpdateRaycasts()
+			{
+				var intersectionModule = evr.GetModule<IntersectionModule>();
+				var distance = k_DefaultRayLength * Viewer.GetViewerScale();
+				ForEachRayOrigin(rayOrigin => { intersectionModule.UpdateRaycast(rayOrigin, distance); });
+			}
+
 			internal void UpdateDefaultProxyRays()
 			{
 				var inputModule = evr.GetModule<MultipleRayInputModule>();
```

Comment:
Same here; If this is called every frame, then go ahead and cache these modules.